api_tests/users/views/test_user_settings.py

```diff
@@ -141,6 +141,87 @@ def test_exceed_throttle(self, mock_mail, app, user_one, url, payload):
         assert res.status_code == 429
 
 
+@pytest.mark.django_db
+class TestUserChangePassword:
+
+    @pytest.fixture()
+    def user_one(self):
+        user = AuthUserFactory()
```

Comment:
Since you're manually using set_password and setting user.auth, you can use `UserFactory` instead of `AuthFactory` to avoid setting the password twice (setting the password is slow).